.github/workflows/gh-page.yml

```
@@ -0,0 +1,35 @@
+name: main
+
+on:
+  push:
+    branches:
+      - "development"
+      - "add-obsolete-action-command"
+  
+jobs:
+  publish-gh-pages:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: true
+
+    - name: Install Dependency
+      working-directory: ./Dashboard
+      run: yarn
+
+    - name: Build .env
+      run: |
+        echo "VITE_OBSOLETE_DATA_PATH=$VITE_OBSOLETE_DATA_PATH" >> ./Dashboard/.env.production
+        echo "VITE_API_PATH=$VITE_API_PATH" >> ./Dashboard/.env.production
```

I don't remember it well... To use `secrets`, it should be:
```suggestion
        echo "VITE_OBSOLETE_DATA_PATH=${{ secrets.VITE_OBSOLETE_DATA_PATH }}" >> ./Dashboard/.env.production
        echo "VITE_API_PATH=${{ secrets.VITE_API_PATH }}" >> ./Dashboard/.env.production
```

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-secrets